ateam_kenobi/src/motion/motion_controller.cpp

```diff
@@ -0,0 +1,195 @@
+// Copyright 2023 A Team
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+#include "motion_controller.hpp"
+
+#include <angles/angles.h>
+#include <CGAL/squared_distance_2.h>
+#include <algorithm>
+#include <cmath>
+#include <vector>
+#include <ateam_msgs/msg/robot_motion_command.hpp>
+#include <ateam_msgs/msg/robot_state.hpp>
+#include <ateam_common/parameters.hpp>
+#include "ateam_geometry/ateam_geometry.hpp"
+#include "control_toolbox/pid.hpp"
+
+/*
+// PID gains
+CREATE_PARAM(double, "motion/pid/x_kp", x_kp, 2);
+CREATE_PARAM(double, "motion/pid/y_kp", y_kp, 2);
+CREATE_PARAM(double, "motion/pid/t_kp", t_kp, 3);
+
+// PID velocity limits
+CREATE_PARAM(double, "motion/pid/x_max", x_max, 2);
+CREATE_PARAM(double, "motion/pid/y_max", y_max, 2);
+CREATE_PARAM(double, "motion/pid/t_max", t_max, 4);
+*/
```

Comment:
Do we still need these commented out PID values?

Reply:
probably not